src/components/Todo/TodoInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
```

Comment:
if you didn't know you can import classNames as cn and use the short form in the code